src/app/CMakeLists.txt

```diff
@@ -2,23 +2,23 @@
 # -----------------------------------------------------------------------------
 # -----------------------------------------------------------------------------
 # Based on https://gist.github.com/giraldeau/546ba5512a74dfe9d8ea0862d66db412
-file(GLOB QBT_TS_FILES "${qBittorrent_SOURCE_DIR}/src/lang/*.ts")
-set_source_files_properties(${QBT_TS_FILES} PROPERTIES OUTPUT_LOCATION "${qBittorrent_BINARY_DIR}/src/lang")
-qt5_add_translation(QBT_QM_FILES ${QBT_TS_FILES} OPTIONS -silent)
+file(GLOB QBT_TS_FILES_LIST "${qBittorrent_SOURCE_DIR}/src/lang/*.ts")
```

Comment:
@Chocobo1

I'm also not a fan of encoding the variable type into its name so explicitly (it's an anti-pattern in most languages), but I feel it is necessary with CMake, where everything is ultimately just a string.

IMO it better conveys the intention and there is a benefit to readability and consistency: now, all other non-list variables are quoted when expanded, whereas this is never the case for list variables. When one is looking at the code now, there is no doubt whether something is a list or not, and whether it's expansion should be quoted or not (remember that in CMake, `someFunction(${SOME_LIST})` will call `someFunction` with each element of `SOME_LIST` as a separate argument).

This change was made for the same reason it is preferable to first declare a variable that's intended to be a list with `list(APPEND SOME_LIST "foo" "bar" "baz")` instead of plain `set` (as in `set(SOME_LIST "foo" "bar" "baz"`), and also to use `list(APPEND ...)` rather than `set(...)` to append elements.

Reply:
@Chocobo1 any more feedback about this? I lumped this change together with the quoting (see the commit message and body), do you have anything against it?